.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "arrowParens": "always",
+  "tabWidth": 2,
+  "singleQuote": false,
+}
```

```suggestion
}

```
There should be a new line at the end of each file.